public/resources/remix/index.js

```diff
@@ -6,122 +6,121 @@
+    var detailsBar = document.querySelector(".details-bar");
+    
+    if (!detailsBar) {
+      return;
     }
 
-    detailsBar.on("click", ".thimble-button",function(){
-      detailsBar.removeClass("collapsed");
-      return false;
+    detailsBar.setAttribute("style", "");
+    detailsBar.classList.add("mouse-mode");
+
+    detailsBar.addEventListener("click", function(event) {
+      if (event.target.classList.contains("thimble-button")) {
+        detailsBar.classList.remove("collapsed");
+        event.stopPropagation();
+      }
     });
 
-    detailsBar.on("click", ".close-details-bar", function(){
-      detailsBar.addClass("collapsed");
-      return false;
+    detailsBar.addEventListener("click", function(event) {
+      if (event.target.classList.contains("close-details-bar")) {
+        detailsBar.classList.add("collapsed");
+        event.stopPropagation();
+      }
     });
 
-    $(".mouse-mode").on("mouseenter",function(){
-      detailsBar.removeClass("collapsed");
+    document.querySelector(".mouse-mode").addEventListener("mouseenter", function(){
+      detailsBar.classList.remove("collapsed");
     });
 
-    $(".mouse-mode").on("mouseleave",function(){
-      detailsBar.addClass("collapsed");
+    document.querySelector(".mouse-mode").addEventListener("mouseleave", function(){
+      detailsBar.classList.add("collapsed");
     });
   }
 
-  function injectDetailsBar($, metadata, callback) {
-    $.ajax({
-      url: metadata.host + "/projects/remix-bar",
-      cache: false,
-      accepts: "text/html",
-      dataType: "html",
-      data: {
-        host: metadata.host,
-        id: metadata.projectId,
-        title: metadata.projectTitle,
-        author: metadata.projectAuthor,
-        updated: metadata.dateUpdated
+  function injectDetailsBar(metadata, callback) {
+    var request = new XMLHttpRequest();
+    var url = metadata.host + "/projects/remix-bar";
+    var data = {
+      host: metadata.host,
+      id: metadata.projectId,
+      title: metadata.projectTitle,
+      author: metadata.projectAuthor,
+      updated: metadata.dateUpdated
+    };
+
+    request.onreadystatechange = function() {
+      if (request.readyState === XMLHttpRequest.DONE ) {
+        var response = request.response;
+        if (request.status !== 200) {
+          return callback({ status: request.status, response: response });
+        }
+        document.querySelector("body").insertAdjacentHTML("afterbegin", response);
+        callback(null);
       }
-    })
-    .done(function(response) {
-      $(response).prependTo("body");
-      callback(null, $);
-    })
-    .fail(function(jqXHR, textStatus) {
-      callback(textStatus);
-    });
+    };
+
+    var querystring = Object.keys(data).map(function(key) {
+      return encodeURIComponent(key) + "=" + encodeURIComponent(data[key]);
+    }).join("&");
+    url = url +  "?" +querystring;
+    request.open("GET", url);
+    request.send(data);
```

Comment:
@aayushsanghavi what browser are you using? This is still incorrect. You shouldn't be sending data to send if it isn't in a valid form (in fact for GET requests you shouldn't be sending request bodies at all).

Take a look at https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Using_XMLHttpRequest which notes this. If you think that is incorrect, you might have another bug in here. Otherwise, you should link to documentation that says that you need to send an object to `send` with GET requests.